airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/spec.json

```diff
@@ -29,12 +29,12 @@
               "client_id": {
                 "title": "Client ID",
                 "type": "string",
-                "description": "The Client ID of your application"
+                "description": "The Client ID of your Lever Hiring developer application"
```

Comment:
"description": "The client ID of your Lever Hiring developer application"
 
 similar with the next descriptions changed "client" word to lowercase.